joplin/base/models.py

```diff
@@ -91,9 +89,7 @@ class ServicePage(Page):
 
     edit_handler = TabbedInterface([
         ObjectList(content_panels, heading='Content'),
-        ObjectList(Page.promote_panels, heading='Promote'),
-        # TODO: What should we do with the fields in settings?
-        # ObjectList(Page.settings_panels, heading='Settings', classname='settings'),
+        ObjectList(Page.promote_panels, heading='SEO'),
```

Comment:
Did we talk about doing this with @toribr? Just wondering bc of our repulsion to using confusing acronyms. "Metadata" or "Page Data" are some more accessible headers I'll throw out there for consideration.

Reply:
We decided on SEO in our initial meeting https://docs.google.com/document/d/1znAEqCRBMJhBuyHJ3KGYUWtqFRTbtub5jMgCcg5qf1k/edit#heading=h.ymikv4aoyttz

> it would be nice to get the promote panel on the same page as in the mocks, but if it needs to be a separate tab/panel it will be labeled SEO 

We didn't talk through it further with content though

Reply:
Talked about this with Content Team and they like the label "Search Info"